### DISCUSSION TITLE
how to create showcase like in the demo?

### DISCUSSION THREAD

**Discussion by twobwithkatt** [HEIGE-PCloud/DoIt#573] [Q&A] [answered]
How can i create a showcase which show in your demo? thanks you first

![image](https://user-images.githubusercontent.com/76549109/163403014-6495cc25-e97f-4e27-9d84-038d1f342f99.png)

**Comment by HEIGE-PCloud** [comment, marked as answer]
https://hugodoit.pages.dev/theme-documentation-extended-shortcodes/#showcase
